web/modules/custom/joinup_core/joinup_core.deploy.php

```diff
@@ -182,3 +185,43 @@ function joinup_core_deploy_0106802(array &$sandbox): string {
 
   return "Processed {$sandbox['progress']} out of {$sandbox['total']}";
 }
+
+/**
+ * Update the EIRA vocabulary.
+ */
+function joinup_core_deploy_0106803(array &$sandbox): void {
+  // Clean up the existing graph.
+  $sparql_connection = Database::getConnection('default', 'sparql_default');
+  $sparql_connection->query('WITH <http://eira_skos> DELETE { ?s ?p ?o } WHERE { ?s ?p ?o } ');
+
+  // Re import the file to update the terms.
+  $connection_options = $sparql_connection->getConnectionOptions();
+  $connect_string = "http://{$connection_options['host']}:{$connection_options['port']}/sparql-graph-crud";
+  $graph_store = new GraphStore($connect_string);
+
+  $filepath = __DIR__ . '/../../../../resources/fixtures/EIRA_SKOS.rdf';
+  $graph = new Graph('http://eira_skos');
+  $graph->parse(file_get_contents($filepath));
+  $graph_store->insert($graph);
```

Comment:
Shouldn't we better `$graph_store->replace($graph);`?

EDIT: I see now that the graph has been emptied before.